### ISSUE TITLE
Install error on Python 3.10.1 windows

### ISSUE THREAD

**Issue by wangxiaoying** [python/typed_ast#183]
Hi,

I encountered [this error](https://github.com/sfu-db/connector-x/runs/4624739492?check_suite_focus=true#step:8:93) when I install type-ast in Python 3.10.1 windows. But when I change it to Python 3.10.0 it [works](https://github.com/sfu-db/connector-x/runs/4624790477?check_suite_focus=true#step:8:91) (installed as `black`'s depenency) .

Please let me know if you need more information. Thank you!

**Comment by srittau** [comment]
From the output it looks like you are trying to install typed-ast 1.4.3. Current version is 1.5.1, did you try that?

**Comment by wangxiaoying** [comment]
Hi @srittau , I'm using poetry to manage my dependencies so I tried to update `black`, seems like the error is gone since `typed-ast` is removed from the newest release. Anyway thanks for the reply!

**Comment by scottshambaugh** [comment]
I ran into the same thing, I'm seeing this issue pop up on CI for python 3.10.1 on Windows 10 and Mac OSX, without any code changes to my repo. `typed-ast 1.4.3` trying to build from source, and building fails for those two configurations. Python 3.7-3.9 work fine, and Ubuntu works fine.

Here's a representative run which was passing about a month ago, but failing now: https://github.com/scottshambaugh/monaco/runs/4847920477?check_suite_focus=true

Edit: This was resolved by unpinning my mypy version from 0.800. More recent versions of mypy removed typed-ast as a dependency.
